packages/manager/apps/pci-workflow/src/setupTests.tsx

```diff
@@ -10,3 +11,8 @@ vi.mock('react-i18next', () => ({
     },
   }),
 }));
+
+const queryClient = new QueryClient();
+export const wrapper = ({ children }) => (
+  <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+);
```

Comment:
[setuPTest](https://create-react-app.dev/docs/running-tests/#initializing-test-environment)
 i dont find it right to export a `wrapper` in here